src/lang/en.json

```diff
@@ -304,7 +306,8 @@
       "NameInvalid": "First name, last name or the username contains invalid characters.",
       "TitleTooLong": "Title can not be longer than 200 characters.",
       "TitleOrDescriptionTooShort": "Title or description can not be shorter than 6 characters.",
-      "Forbidden": "Access denied"
+      "Forbidden": "You are trying to access forbidden resources",
```

Comment:
"Forbidden" is the same as "accessDenied". Maybe we can compress it to one item?